app/controllers/events_controller.rb

```diff
@@ -4,6 +4,10 @@ class EventsController < InheritedResources::Base
   skip_load_and_authorize_resource only: [:index, :show]
   before_filter :set_types
 
+  def participants
+    @breadcrumbs = [{ label: 'events', url: events_path }, { label: @event.name, url: event_path(@event) }, { label: 'participants' }]
```

Comment:
Of course a matter of taste. I would prefer this format:

```ruby
@breadcrumbs = [
  { label: 'events', url: events_path },
  { label: @event.name, url: event_path(@event) },
  { label: 'participants' }
] 
```

@ChrisBr @nikhilgupta1211  What do you say?